### DISCUSSION TITLE
Default language

### DISCUSSION THREAD

**Discussion by loan-talvat** [umami-software/umami#808] [Ideas]
Is it possible to define in the .ENV or in the parameters a default language, or at least to detect the language of the browser ?

**Comment by timonson** [comment]
Does anybody have a good workaround in the meantime?

**Comment by Maxime-J** [comment]
Didn't tried it yet, but there's a `DEFAULT_LOCALE` constant in [lib/constants.js](https://github.com/mikecao/umami/blob/master/lib/constants.js)
Setting it to the wanted default locale before build might work.

**Reply by Maxime-J** [reply]
Just a confirmation it works :)
